http/config.go

```diff
@@ -93,6 +93,7 @@ type AuthConfig struct {
 	// Type specifies the type of authentication required for the interface.
 	Type AuthType `koanf:"type"`
 	AuthorizedKeysPath string `koanf:"authorizedkeyspath"`
+	Audience string `koanf:"audience"`
```

Comment:
I assume this is here to prevent JWTs, stolen from somewhere else, to be re-used to authenticate access to the Nuts node?

Is it required to be configured?

Reply:
It's indeed effectively a sort of hostname configuration. It's not required to be configured as it defaults to the machine hostname. In many cases the hostname of a linux box is not the same as the well known DNS name, and thus this configuration value will help administrators configure the system in a way that makes sense to end users but also works with their underlying platform which sometimes issues random hostnames. TLS servers have a similar issue where the hostname is not always a reliable source of the well known hostname/DNS name of a server